.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,57 @@
+## Developer
+
+Write a brief summary of changes here, but do not repeat commit messages (which
+should have a more complete description).
+
+
+### Tickets affected
+
+- https://mitlibraries.atlassian.net/browse/
+
+### Version (see config/theme.ini)
+
+- [ ] The theme's version number has been incremented, setting up a production
+      deploy.
+
+### Documentation
+
+- [ ] Project documentation has been updated.
+- [ ] No documentation changes are needed.
+
+### Accessibility
+
+- [ ] ANDI or WAVE has been run in accordance to [our guide](https://mitlibraries.github.io/guides/basics/a11y.html).
+- [ ] New flagged issues not already resolved in this PR have been ticketed 
+      (link in the Pull Request details above).
+- [ ] This PR contains no changes to the view layer.
+
+### Stakeholder approval
+
+- [ ] Stakeholder approval has been confirmed (see ticket).
+- [ ] Stakeholder approval will happen retroactively.
+- [ ] Stakeholder approval is not needed.
+
+### Dependencies
+
+YES | NO dependencies are updated
+
+### Additional context needed to review
+
+E.g., if the PR includes updated dependencies and/or data
+migration, or how to confirm the feature is working.
+
+
+## Code Reviewer
+
+##### Code
+
+- [ ] I have confirmed that the code works as intended.
+- [ ] Any CodeClimate issues have been fixed or confirmed as
+added technical debt.
+
```

Comment:
Consider added:

```
- [ ] I have reviewed any new dependencies and agree they are appropriate
```

Reply:
Oh. Never mind probably. I think that's covered in the next section I hadn't gotten to. I kinda like that better in the code section than docs though but either is fine.